client/src/templates/Introduction/components/block.css

```diff
@@ -0,0 +1,15 @@
+.big-block-title > svg {
+  height: 1.5rem;
+  display: inline;
+  width: auto;
+  margin-right: 10px;
+  vertical-align: text-bottom;
```

Comment:
```suggestion
  height: 1.6rem;
  display: inline;
  width: auto;
  margin-right: 10px;
  vertical-align: bottom;
```
```suggestion
  vertical-align: bottom;
```
bottom:
![image](https://user-images.githubusercontent.com/15801806/149761248-9419b3fa-ffd6-4e3c-b16b-2a73a85ef71a.png)

text-bottom:
![image](https://user-images.githubusercontent.com/15801806/149761317-9284ab9b-a5a3-4d60-8245-cac68f67a89c.png)

bottom + height: 1.6rem:
![image](https://user-images.githubusercontent.com/15801806/149761605-1258b885-c584-4720-a9af-8db09c109b04.png)

I think the 3rd's the prettiest.